building-and-packaging/material/packaging_python_slides.md

```diff
@@ -63,6 +63,7 @@ Note: `<directory-name>` should have a folder with `<package-name>`
 - Before doing installing with `setup.py` you have to manually search and install dependencies.
 - A package installed by `setup.py` needs to be manually maintained and uninstalled if required.
 - Requires manual downloading of files from the package website.
+- Packages cannot be easily shared between projects, so you'd have to manually install the package in two projects.
```

Comment:
I am curious about this point:

- Could you maybe elaborate a bit on the meaning? 
- In what sense am I not able to share the package between projects? 
  - Would it not be a possibility to install the package once in a central location on my system that is accessible from other projects/codes?

Reply:
of course you can share the packages between projects by putting them in a known location.

when i read "manually search and install dependencies" was downloading the source of a package into your project root and installing it there instead of globally, so maybe i misunderstood this one

you'd have to either put the packages in a global location (and hope there are no conflicts) or you need to symlink them into the working directory of the project which is kinda bad in many cases if your code might not be able to dereference a symlink.

if you *would* have multiple packages downloaded in multiple project dirs, you'd of course have an additional overhead keeping them up to date

so i guess i can remove this as i simply mistook @IshaanDesai during the lecture

Reply:
No, don't worry. I think you have a valid point. To me it was just not completely clear what you meant. :smile: 

Maybe the phrasing is not good on the slides yet. I guess @IshaanDesai knows best.

Reply:
@cheriimoya I think the point you have added is very valid. I will just restructure it a bit before merging. You have formulated the sentence as `cannot be easily shared ...` which is good because that means that it is not impossible, just cumbersome and hence something to avoid.